projects/cashmere/src/lib/app-switcher/app-switcher.service.ts

```diff
@@ -1,31 +1,23 @@
-import {Observable, of} from 'rxjs';
+import {Observable} from 'rxjs';
 
-import {delay} from 'rxjs/operators';
 import {HttpClient} from '@angular/common/http';
 import {Inject, Injectable} from '@angular/core';
-import {getMockApplication} from './app-switcher-mock';
-import {IAppSwitcherConfig, IAppSwitcherService, IDiscoveryRequest} from './app-switcher-interfaces';
-
-export class MockAppSwitcherService implements IAppSwitcherService {
-    public readonly allApplicationsUri: string = '#';
-
-    constructor() {}
-
-    public getApplications(): Observable<IDiscoveryRequest> {
-        return of({value: getMockApplication(5)}).pipe(delay(500));
-    }
-}
+import {IAppSwitcherConfig, IAppSwitcherService, IDiscoveryRequest, APP_SWITCHER_CONFIG} from './app-switcher-interfaces';
 
 @Injectable()
 export class AppSwitcherService implements IAppSwitcherService {
```

Comment:
Don't have to change this, but I was thinking, would it be better to call this `DiscoveryAppSwitcherService`?  It would be more clear the coupling between the DiscoveryService and this service this way, and not cause naming conflicts if there is a custom app switcher service not dependent on DiscoveryService.  Just my 2¢.

Reply:
I like this, I think I want to focus on more important issues, so feel free to open an issue